test/LockedNORI.test.ts

```diff
@@ -988,12 +1010,18 @@ describe('LockedNori', () => {
         )
         .to.emit(bpNori, 'Transfer')
         .withArgs(lNori.address, namedAccounts.admin, quantityRevoked);
+
+      await hardhat.network.provider.send('evm_setNextBlockTimestamp', [
+        grant.startTime + VEST_REVOKED_OFFSET,
+      ]);
+      await hardhat.network.provider.send('evm_mine');
+
       expect(await lNori.vestedBalanceOf(employee)).to.equal(newBalance);
       expect(await lNori.balanceOf(employee)).to.equal(newBalance);
 
       expect(await lNori.totalSupply()).to.eq(newBalance);
       expect(await lNori.unlockedBalanceOf(employee)).to.eq(
-        BigNumber.from('600008888888888888888')
+        BigNumber.from('600000000000000000000')
```

Comment:
for consistency, we should prefer
```suggestion
        formatTokenAmount(600)
```
though I realize `Bignumber.from` is also being used in several places so this doesn't have to happen for this PR